src/pages/docs/concepts/architecture-options.md

```diff
@@ -0,0 +1,138 @@
+# Architecture options
+
+Possible architectures of using web3.storage to upload
+
+UCAN opens up a number of options in how to integrate with w3up: Should you, the developer, own the Space? Should you delegate permissions to your users? Or should your user own their own Space? Broadly, there are three ways to integrate:
+
+- Client-server: You (the developer) own the Space, and your user uploads to your backend infra before you upload it to the service
+- Delegated: You own the Space, but you give a delegated UCAN token to your user's Agent to upload directly to the service (rather than needing to touch the upload in your backend)
```

Comment:
```suggestion
- Delegated: You own the Space, but you give a delegated UCAN token to your user's Agent to upload directly to the service, rather than needing to proxy the upload through your backend (no egress from your infrastructure)
```

I think something where also clearly states an advantage is good your. no egress costs to proxy through and needed infra